Makefile

```diff
@@ -4,21 +4,21 @@
 
 ### Generic top-level targets
 
-all: riscv/all sandbox/all jstz/all dummy/all page-cache-tester/all etherlink/all
+all: riscv/all jstz/all dummy/all page-cache-tester/all etherlink/all
 
 build-deps: riscv/build-deps jstz/build-deps etherlink/build-deps
 
 build-deps-slim: riscv/build-deps-slim
 
-check: riscv/check sandbox/check jstz/check dummy/check page-cache-tester/all etherlink/check assets/check
+check: riscv/check jstz/check dummy/check page-cache-tester/check etherlink/check assets/check
 
-build: sandbox/build jstz/build dummy/build page-cache-tester/build etherlink/build
+build: riscv/build jstz/build dummy/build page-cache-tester/build etherlink/build
```

Comment:
There is nothing to build of value using `riscv/build`, why spend time running it?